docs/collections/_docs/really-using-getnighthawk.md

```diff
@@ -1,196 +1,7 @@
-
-**OutputCollector** is a container that facilitates building up the native output
-format of Nighthawk (`proto3`, `nighthawk::client::Output`). It is the basis for all
-output formats offered by Nighthawk, including CLI human output.
-
-### OutputFormatter
-
-**OutputFormatter** is responsible for transformations of `nighthawk::client::Output`
-to requested formats (e.g. human, json, fortio, etc)
-
-### Statistic
-
-Nighthawk’s **Statistic** is responsible for administrating latencies. The most
-notable implementation that exists today wraps
-[HdrHistogram](https://github.com/HdrHistogram/HdrHistogram_c), but Nighthawk
-also has a couple of other implementations which mostly exist to ensure that
-floating point math is correct in tests, as well as a simple efficient
-implementation that simply tracks the `mean` and `pstddev` for those cases where
-we don't need percentiles. For various reasons, HdrHistogram might get replaced
-by [libcirclhist](https://github.com/envoyproxy/nighthawk/issues/115) in the
-near future.
-
-### H1Pool & H2Pool
-
-Nighthawk derives its own version of these from the vanilla Envoy ones. It does
-that to implement things like pro-active connection pre-fetching and H2
-multi-connection support, as well as offer more connection management
-strategies.
-
-## Nighthawk binaries
-
-### nighthawk_client
-
-The CLI interface of the Nighthawk client. It synthesizes traffic according
-to the requested configuration and report results in the requested output format.
-
-### nighthawk_service
-
-Nighthawk’s gRPC service is able to execute load tests, and report results.
-Under the hood it shares much of the code of nighthawk_client, and effectively
-it allows to efficiently perform remote back-to-back executions of that.
-
-### nighthawk_test_server
-
-Nighthawk’s test server, based on Envoy. It is able to synthesize delays and
-responses based on configuration via request headers (next to on-disk
-configuration).
-
-### nighthawk_output_transform
-
-Utility for transforming the nighthawk-native json output format into
-other formats (e.g. human, fortio). It can be very useful to always store the
-json output format, yet be able to easily get to one of the other output
-formats. It’s like having the cake and eating it too!
-
-## User-specified Nighthawk logging
-
-Users of Nighthawk can specify custom format and destination (logging sink
-delegate) for all Nighthawk logging messages. Nighthawk utilizes the Envoy's
-logging mechanism by performing all logging via the **ENVOY_LOG** macro. To
-customize this mechanism, users need to perform two steps:
-1. Create a logging sink delegate inherited from [Envoy SinkDelegate](https://github.com/envoyproxy/envoy/blob/main/source/common/common/logger.h).
-2. Construct a ServiceImpl object with an [Envoy Logger Context](https://github.com/envoyproxy/envoy/blob/main/source/common/common/logger.h) which contains user-specified log level and format.
\ No newline at end of file
+Test section 1
```

Comment:
@vinayaksh42 it would be good to transition the deleted text into a section (or two) if we can.

Reply:
sure